docs/manual/common/guide/production/code/conductr.sbt

```diff
@@ -0,0 +1,3 @@
+//#sbt-conductr
+addSbtPlugin("com.lightbend.conductr" % "sbt-conductr" % "2.1.17")
```

Comment:
Yes, and it will need to be updated again once we actually add Lagom Scala support to sbt-conductr, which is why I didn't bother updating the version.